src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -141,6 +142,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <_ExtraTrimmerArgs>--skip-unresolved true $(_ExtraTrimmerArgs)</_ExtraTrimmerArgs>
       <TrimMode Condition=" '$(TrimMode)' == '' ">copyused</TrimMode>
+      <_TrimmerLinkSymbols Condition=" '$(_TrimmerLinkSymbols)' == '' ">$(DebuggerSupport)</_TrimmerLinkSymbols>
+      <_TrimmerLinkSymbols Condition=" '$(_TrimmerLinkSymbols)' == '' ">true</_TrimmerLinkSymbols>
```

Comment:
Do we actually need this property at all?

Reply:
We need some way to set the linker option to `true` when `DebuggerSupport` is empty. We could do that by defaulting `DebuggerSupport` itself to `true`, but I don't want to commit to making that default available at a particular place in the import chain - it would have to happen before `RuntimeHostConfigurationOption`, but if it happens too early, then SDK components won't be able to distinguish between the default `true` and an explicit `true` set by the user.

Aside from that, this provides an "escape hatch" for people who want to tweak just the linker settings and are ok with relying on an unsupported property to experiment with things - see @eerhardt's [comment](https://github.com/dotnet/sdk/pull/12144#discussion_r446146454) for example.